python-backend/app/api/now_submissions/resources/application_list_resource.py

```diff
@@ -20,14 +20,24 @@ class ApplicationListResource(Resource, UserMixin, ErrorMixin):
         description='Get a list of applications. Order: receiveddate DESC',
         params={
             'page': f'The page number of paginated records to return. Default: {PAGE_DEFAULT}',
-            'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}'
+            'per_page': f'The number of records to return per page. Default: {PER_PAGE_DEFAULT}',
+            'status': 'Comma-separated list of statuses to include in results. Default: All statuses.',
+            'noticeofworktype': 'Substring to match with a NoW\s type',
```

Comment:
is there a reason we don't follow the convention notice_of_work_type like we do for other multy word filters?

Reply:
Because this is the column name. It was pulled over from NROS

Reply:
fair enough